code/modules/clothing/suits/armor.dm

```diff
@@ -391,16 +398,18 @@
 	desc = "A standard armor vest fielded for SolGov's infantry."
 	icon_state = "armor_solgov"
 	item_state = "armor_solgov"
+	armor = list("melee" = 15, "bullet" = 60, "laser" = 10, "energy" = 10, "bomb" = 40, "bio" = 0, "rad" = 0, "fire" = 50, "acid" = 50)
```

Comment:
if you're going to give it bulletproof armor stats subtype it under the bulletproof vest